packages/shared/src/components/modals/ReportModal.module.css

```diff
@@ -0,0 +1,27 @@
+.reportModal {
+  & :global(.overlay) {
+    justify-content: center;
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
+  }
+
+  & :global(.modal) {
+    max-width: 23.25rem;
+    padding: 2rem;
+  }
+}
+
+.buttons {
+  & > button {
+    flex: 1;
+    margin: 0 0.5rem;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
```

Comment:
This can be solved with Tailwind classes I think. This should be our first priority only then fall back to CSS modules

Reply:
Ah true, was being lazy as this came from the existing one.
Will refactor you're completely right 👀